docusaurus/docs/dev-docs/api/rest/guides/understanding-populate.md

```diff
@@ -0,0 +1,1653 @@
+
+Populating in the context of the Strapi REST API means including additional content with your response by returning more fields than the ones returned by default. You use the [`populate` parameter](#population) to achieve this.
+
+:::info
+Throughout this guide, examples are built with real data queried from the server included with the [FoodAdvisor](https://github.com/strapi/foodadvisor) example application. To test examples by yourself, setup FoodAdvisor, start the server in the `/api/` folder, and ensure that proper `find` permissions are given for the queried content-types before sending your queries.
+:::
+
+The present guide will cover detailed explanations for the following use cases:
+
+- populate [all fields and relations, 1 level deep](#populate-all-relations-and-fields-1-level-deep),
+- populate [some fields and relations, 1 level deep](#populate-1-level-deep-for-specific-relations),
+- populate [some fields and relations, several levels deep](#populate-several-levels-deep-for-specific-relations),
+- populate [components](#populate-components),
+- populate [dynamic zones](#populate-dynamic-zones).
+
+:::strapi Advanced use case: Populating creator fields
+In addition to the various ways of using the `populate` parameter in your queries, you can also build a custom controller as a workaround to populate creator fields (e.g., `createdBy` and `updatedBy`). This is explained in the dedicated [How to populate creator fields](/dev-docs/api/rest/guides/populate-creator-fields) guide.
+:::
+
+## Populate all relations and fields, 1 level deep
+
+You can return all relations, media fields, components and dynamic zones with a single query. For relations, this will only work 1 level deep, to prevent performance issues and long response times.
+
+To populate everything 1 level deep, add the `populate=*` parameter to your query.
+
+The following diagram compares data returned by the [FoodAdvisor](https://github.com/strapi/foodadvisor) example application with and without populating everything 1 level deep:
+
+![Diagram with populate use cases with FoodAdvisor data ](/img/assets/rest-api/populate-foodadvisor-diagram1.png)
+
+Let's compare and explain what happens with and without this query parameter:
+
+<SideBySideContainer>
+<SideBySideColumn>
+
+**Without `populate`**
+
+Without the populate parameter, a `GET` request to `/api/articles` only returns the default attributes and does not return any media fields, relations, components or dynamic zones.
+
+The following example is the full response for all 4 entries from the `articles` content-types.
+
+Notice how the response only includes the `title`, `slug`, `createdAt`, `updatedAt`, `publishedAt`, and `locale` fields, and the field content of the article as handled by the CKEditor plugin (`ckeditor_content`, truncated for brevity):
```

Comment:
> truncated for brevity

I've seen this so much more recently that I laughed a little that it's in our docs :laughing: :+1: